tests/ui/type-alias-impl-trait/issue-70121.rs

```diff
@@ -1,5 +1,3 @@
-// check-pass
-
 #![feature(type_alias_impl_trait)]
```

Comment:
known-bug here?

Reply:
it's on purpose at this point, I don't know if we want to fix it at all. It's more of a missing feature than a bug.

Reply:
ok, just curious if we want to "track" that somehow, though you can make the call here for if it's needed and exactly how

Reply:
This is information that goes into the stabilization doc about what exactly we are stabilizing.